<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pievienot_kontaktinfo</class>
 <widget class="QDialog" name="pievienot_kontaktinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pievienot kontaktinformāciju</string>
  </property>
  <widget class="QComboBox" name="pers_ID">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dzimsanas_datums">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>5</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Persona (obligats)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Līgums(obligats)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Amats_ID">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adrese(obligats)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Adrese">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefons nummurs</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Kontaktinformācija</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzimšanas datums(obligats)</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="lig_ID">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pievienot_kontakt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>pievienot</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tel_nr">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Amats(obligats)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
